spec/integration/schedule_spec.rb

```diff
@@ -8,13 +8,15 @@
 require 'gooddata/helpers/global_helpers'
 
 describe GoodData::Schedule do
-  SCHEDULE_ID = ScheduleHelper::SCHEDULE_ID
-  SCHEDULE_URL = "/gdc/projects/#{ProjectHelper::PROJECT_ID}/schedules/#{SCHEDULE_ID}"
-
-  before(:each) do
+  before(:all) do
```

Comment:
error: Identical code found in schedule_spec.rb:11, schedule_spec.rb:11 (mass = 128) (flay)